.gitignore

```
@@ -35,3 +35,4 @@ test-results
 .cache/
 .eslintcache
 playwright.env
+firefox
```

Sorry that I removed that, but reverting it will not be helpful given that it will ignore files from some sub folders that contain `firefox`. So this should only apply for the top-level folder then:

```suggestion
/firefox
```

---

Maybe we should instead put it into a temporary directory? /tmp

---

Changed to `/firefox/` for now.